less/core/tooltip.less

```diff
@@ -0,0 +1,25 @@
+.tooltip {
+  display: none;
+  position: absolute;
+  top:0;
+  left:0;
+  background-color: black;
+  color: white;
+  overflow: visible;
+  min-width: 100px;
```

Comment:
should we use rem rather than px?

Reply:
Hey Eleanor, thanks for your feedback on this, I've refactored most of this, Ill push it up next week but will make sure I capture all your comments above!